pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -612,6 +612,10 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   });
 
+  noice-nvim = super.noice-nvim.overrideAttrs(old: {
+    dependencies = with self; [ nui-nvim nvim-notify nvim-cmp ];
```

Comment:
not sure but I think nvim-cmp has become optional in the latest release ?

Reply:
I think ur right